experimental/examples/ConfigExample.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../../vendor/autoload.php';
+
+use OpenTelemetry\Experimental\Config\ConfigBuilder;
+use OpenTelemetry\Experimental\Config\Exporter\NewRelic\Config as NewRelicConfig;
+use OpenTelemetry\Experimental\Config\Exporter\Zipkin\Config as ZipkinConfig;
+
+//@see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md
+putenv('OTEL_RESOURCE_ATTRIBUTES=service.version=1.0.0,service.namespace=my_app');
+putenv('OTEL_SERVICE_NAME=example-app');
+putenv('OTEL_LOG_LEVEL=warning');
+putenv('OTEL_TRACES_SAMPLER=traceidratio');
+putenv('OTEL_TRACES_SAMPLER_ARG=0.95');
+putenv('OTEL_TRACES_EXPORTER=otlp,zipkin,newrelic');
+putenv('OTEL_EXPORTER_ZIPKIN_ENDPOINT=http:zipkin/v1/traces');
+putenv('OTEL_EXPORTER_ZIPKIN_TIMEOUT=99');
+putenv('OTEL_PHP_TRACES_PROCESSOR=batch,simple,noop');
+putenv('OTEL_BSP_SCHEDULE_DELAY=10000');
+putenv('OTEL_ATTRIBUTE_COUNT_LIMIT=111');
+putenv('OTEL_PROPAGATORS=tracecontext');
+
+echo 'Creating Config From Environment and user config' . PHP_EOL;
+$config = (new ConfigBuilder())
+    ->withExporterConfig(ZipkinConfig::class)
```

Comment:
What's the point of expecting a class, when this has to be added programmatically anyway?

Reply:
My thinking here is to allow making contrib exporters available without directly referencing `contrib` from `sdk`, and potentially to make them settable from an env var (OTLP_PHP_EXPORTER_CLASS ?).
Without some sort of discovery mechanism I don't see a way to have something in `sdk` that is capable of configuring classes from `contrib`. Perhaps a `class_exists` for something like `Contrib\ExporterBuilder`, and the configurations for these guys actually does live in `contrib`, and if that class does not exist then only the SDK SpanExporters are available.

Reply:
Ok, got you, I just find the constructor a bit clunky in that case. But I think you commented something regarding this below.